lib/ecto/migration.ex

```diff
@@ -1549,7 +1549,10 @@ defmodule Ecto.Migration do
     struct!(%Constraint{table: table, name: name}, opts)
   end
 
-  @doc "Executes queue migration commands."
+  @doc """
+  Execute all changes specified by the migration so far.
```

Comment:
```suggestion
  Execute all changes specified by the migration so far.

```

Reply:
We should keep the first paragraph succinct as that is used as summary in docs. :)